README.md

```diff
@@ -6,13 +6,13 @@
 
 An extension to the [Flutter SDK](https://github.com/flutter/flutter) for building Flutter applications for Tizen devices.
 
-_Flutter and the related logo are trademarks of Google LLC. We are not endorsed by or affiliated with Google LLC._
+_Flutter and the related logo are trademarks of Google LLC. This project is sponsored and maintained by Samsung Research._
 
 ## Supported devices
 
-- **Galaxy Watch**: Tizen 4.0 (2018) or later
-- **Smart TV**: [Tizen 6.0 (2021) or later](https://developer.samsung.com/smarttv/develop/specifications/tv-model-groups.html) (older models are not supported due to [security reasons](https://github.com/flutter-tizen/flutter-tizen/wiki/Limitations#tv-limitations))
-- **IoT (RPi 3/4)**: Tizen 5.5 (2020) or later
+- **Galaxy Watch**: Tizen 5.5 or later
```

Comment:
Does the landing of this PR mean that Tizen 4.0 is not supported?

Reply:
Not _officially_ supported. This means that we may remove support from our codebase at any time without additional notice.